ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -210,6 +219,20 @@ const Footer = ({ isFormValid, setStepNotification }) => {
                                   onClick={() => goToStep(activeStep, onNext)}>
                                     {nextButtonText}
                                 </Button>
+                                <Tooltip
+                                  id="next-tooltip-ref"
+                                  content={
+                                      <div>
+                                          {_("To continue, select the devices(s) to install to.")}
+                                      </div>
+                                  }
+                                  // Only show the tooltip on installation destination spoke that is not valid (no disks selected).
+                                  // NOTE: As PatternFly Button with isDisabled set apprently does not get any mouse events anymore,
+                                  //       we need to manually trigger the tooltip.
+                                  reference={() => document.getElementById("installation-next-btn")}
+                                  trigger="manual"
+                                  isVisible={activeStep.id === "installation-destination" && !isFormValid}
```

Comment:
Don't you want to actually skip the `Tooltip` component if this is a different step?